iac/tf-multienv-cicd-anthos-autopilot/README.md

```diff
@@ -0,0 +1,49 @@
+# What does this solution contain?
```

Comment:
Even before this, I'd give a disclaimer saying that this entire TF directory is not meant for everyday usage and is _not_ required to deploy Bank of Anthos or any of its extensions / flavours. It's meant to showcase how you could do a full CI/CD environment, etc etc...

Reply:
Done.